website/docs/routes/programming.mdx

```diff
@@ -474,6 +474,10 @@ GitHub provides some official RSS feeds:
 
 <Route namespace="rustcc" data={{"path":"/jobs","categories":["programming"],"example":"/rustcc/jobs","parameters":{},"features":{"requireConfig":false,"requirePuppeteer":false,"antiCrawler":false,"supportBT":false,"supportPodcast":false,"supportScihub":false},"radar":{"source":["rustcc.cn/"]},"name":"招聘","maintainers":["CcccFz"],"url":"rustcc.cn/","location":"jobs.ts"}} />
 
+### 新闻/聚合
+
+<Route namespace="rustcc" data={{"path":"/news","categories":["programming"],"example":"/rustcc/news","parameters":{},"features":{"requireConfig":false,"requirePuppeteer":false,"antiCrawler":false,"supportBT":false,"supportPodcast":false,"supportScihub":false},"radar":{"source":["rustcc.cn/"]},"name":"新闻/聚合","maintainers":["zhenlohuang"],"url":"rustcc.cn/","location":"news.ts"}} />
+
```

Comment:
Do not commit any changes to the docs, it will be generated automatically and the changes will be overwritten.